common/windows/delphi/tools/certificates/build.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../../../../../resources/build/builder.inc.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+builder_describe "Build and install test certificates" clean configure build test \
+  "certificates   Create test certificates and install root certificates"
```

Comment:
Why not...

```suggestion
  "certificates+   Create test certificates and install root certificates"
```

It's the only action with an actual effect, so why not make it the default?

Also... I do find it interesting as the parent project doesn't actually trigger anything for this child project.  Granted, there are a _lot_ of moving parts.

----

Also mildly interesting to me:  former `verify` Makefile targets among this child project's siblings were converted into `build` actions... but not the `test-certificate` target for _this_ child project, which is now `certificates`.

Reply:
I deliberately don't build the certificates automatically. If the dev wants local test certificates, they should run `./build.sh certificates` to build and install them. The build is destructive in that it overwrites existing certificates in the folder, and then the install adds these test root certificates to the local system, so we want this to be a careful, deliberate action. (And yeah, probably could document this a little more!)

And yeah, did a little bit of cleanup as well. I guess certificates/ doesn't need to be called from the parent build.sh, really.

Reply:
https://github.com/keymanapp/keyman/blob/f6a64bc223491e35cea88ced6936be029a81e3ce/windows/src/README.md#L60-L74 needs updating, will work on that